pandas/tests/indexes/datetimes/test_timezones.py

```diff
@@ -581,12 +581,7 @@ def test_dti_construction_ambiguous_endpoint(self, tz):
             ["US/Pacific", "shift_forward", "2019-03-10 03:00"],
             ["dateutil/US/Pacific", "shift_forward", "2019-03-10 03:00"],
             ["US/Pacific", "shift_backward", "2019-03-10 01:00"],
-            pytest.param(
-                "dateutil/US/Pacific",
-                "shift_backward",
-                "2019-03-10 01:00",
-                marks=pytest.mark.xfail(reason="GH 24329"),
-            ),
+            ["dateutil/US/Pacific", "shift_backward", "2019-03-10 01:00"],
```

Comment:
This shortcut, along with the fix to #24329 makes this exception no longer necessary, as a correct value gets returned without an error.